RunOneSim.py

```diff
@@ -22,14 +22,43 @@
 
 #====================================== HELPER =================================================
 
-def runSim(simSetting, simUI=None):
+def runOneSim(simSetting, simUI=None):
     '''
     Run a single simulation. Finishes when map is complete (or mapping times out).
     '''
 
     # ======================== setup
 
-    log.info("Simulation started")
+    # log
+    log.info(
+        "run {}/{} starting ...".format(
```

Comment:
here, you assume the seed is incrementing, which is not true in the general case. I recommend you simply log "print starting".